.github/actions/send-webhook-notification/action.yml

```diff
@@ -17,7 +17,7 @@ runs:
       run: |
         curl -X POST \
         -H 'Content-Type: application/json' \
-        ${{ inputs.webhook-url }} \
+        '${{ inputs.webhook-url }}' \
```

Comment:
The webhook-test URL is working fine even without quotes but the chat webhook-url is big and contains special characters so doesn't work without the quote.